react-form-controls/my-app/src/RegistrationFormControlled.js

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+
+export default function RegistrationFormControlled() {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    console.log('state:', { username, password });
```

Comment:
No need to wrap in `{}`: `console.log('state:', username, password)` will log the values.